mods/ts/chrome/ingame-player.yaml

```diff
@@ -306,52 +334,44 @@ Container@PLAYER_WIDGETS:
 							HotkeyName: ProductionTypeVehicleKey
 							Children:
 								Image@ICON:
-									X: 6
-									Y: 6
+									Logic: AddRaceSuffixLogic
+									X: 7
+									Y: 7
 									ImageCollection: production-icons
 						ProductionTypeButton@AIRCRAFT:
 							Logic: AddRaceSuffixLogic
-							Y: 124
-							Width: 28
-							Height: 28
+							X: 148
+							Width: 30
+							Height: 31
 							VisualHeight: 0
 							Background: sidebar-button
 							TooltipText: Aircraft
 							TooltipContainer: TOOLTIP_CONTAINER
-							ProductionGroup: Air
+							ProductionGroup: Aircraft
```

Comment:
This breaks things. Please revert it.

![image](https://cloud.githubusercontent.com/assets/756669/6765378/eaf1c5ce-cfdd-11e4-8d62-290bf55c6236.png)